src/ui/public/vislib/visualizations/point_series/column_chart.js

```diff
@@ -10,8 +10,11 @@ export function VislibVisualizationsColumnChartProvider(Private) {
     mode: 'normal',
     showTooltip: true,
     color: undefined,
-    fillColor: undefined
+    fillColor: undefined,
   };
+
+  const minBarWidth = 1;
```

Comment:
@trevan but whats large enough ? currently we set minimum chart width of 50px .... but in this scenario we are actually talking about 1px wide charts (using chart split instead of x - axis) ...
(it also takes considerably more time to render than the x-axis ... however that's a diff problem)